tests/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ex
+export VAGRANT_DEFAULT_PROVIDER=${VAGRANT_DEFAULT_PROVIDER:-libvirt}
+export SSL=true
+export ENCRYPTION=true
+export STATSD_EXPORTER=true
+
+# TEST JAMMY & NOBLE
+for box in 22.04 24.04; do
```

Comment:
is the understanding that libvirt and & vmware seem to support the exact same boxes/tests?  It's only x86-vbox tests that manage to get some coverage of the 20.04 image?

Reply:
ugh, I forgot to publish some comments. The tests didn't all pass on vmware/parallels on arm :(